consensus/polybft/polybft.go

```diff
@@ -222,45 +211,45 @@ func GenesisPostHookFactory(cfg *chain.Chain, engineName string) func(txn *state
 						return err
 					}
 
-					// initialize ChildERC20PredicateAccessList SC
+					// initialize InternalERC20PredicateAccessList SC
 					if err = initERC20ACLPredicateContract(transition, chainBridgeCfg, cfg.Genesis.Alloc,
 						owner, useBridgeAllowList, useBridgeBlockList, false, chainIDBig,
-						"ChildERC20PredicateAccessList"); err != nil {
+						"InternalERC20Predicate"); err != nil {
 						return err
 					}
 
-					// initialize ChildERC721PredicateAccessList SC
+					// initialize InternalERC721PredicateAccessList SC
 					if err = initERC721ACLPredicateContract(transition, chainBridgeCfg, cfg.Genesis.Alloc,
 						owner, useBridgeAllowList, useBridgeBlockList, false, chainIDBig,
-						"ChildERC721PredicateAccessList"); err != nil {
+						"InternalERC721Predicate"); err != nil {
 						return err
 					}
 
-					// initialize ChildERC1155PredicateAccessList SC
+					// initialize InternalERC1155PredicateAccessList SC
 					if err = initERC1155ACLPredicateContract(transition, chainBridgeCfg, cfg.Genesis.Alloc,
 						owner, useBridgeAllowList, useBridgeBlockList, false, chainIDBig,
-						"ChildERC1155PredicateAccessList"); err != nil {
+						"InternalERC1155Predicate"); err != nil {
 						return err
 					}
 
-					// initialize RootMintableERC20PredicateAccessList SC
+					// initialize InternalMintableERC20PredicateAccessList SC
 					if err = initERC20ACLPredicateContract(transition, chainBridgeCfg, cfg.Genesis.Alloc,
 						owner, useBridgeAllowList, useBridgeBlockList, true, chainIDBig,
-						"RootERC20PredicateAccessList"); err != nil {
+						"InternalERC20MintablePredicate"); err != nil {
 						return err
 					}
 
-					// initialize RootMintableERC721PredicateAccessList SC
+					// initialize InternalMintableERC721PredicateAccessList SC
 					if err = initERC721ACLPredicateContract(transition, chainBridgeCfg, cfg.Genesis.Alloc,
 						owner, useBridgeAllowList, useBridgeBlockList, true, chainIDBig,
-						"RootERC721PredicateAccessList"); err != nil {
+						"InternalERC721MintablePredicate"); err != nil {
 						return err
 					}
 
-					// initialize RootMintableERC1155PredicateAccessList SC
+					// initialize InternalMintableERC1155PredicateAccessList SC
 					if err = initERC1155ACLPredicateContract(transition, chainBridgeCfg, cfg.Genesis.Alloc,
 						owner, useBridgeAllowList, useBridgeBlockList, true, chainIDBig,
-						"RootERC1155PredicateAccessList"); err != nil {
+						"InternalERC1155MintablePredicate"); err != nil {
```

Comment:
Review the other places as well
```suggestion
						"InternalMintableERC1155PredicateAccessList"); err != nil {
```